contracts/contracts/strategies/NativeStaking/CompoundingValidatorManager.sol

```diff
@@ -109,13 +109,12 @@ abstract contract CompoundingValidatorManager is Governable, Pausable {
     /// withdrawal of the validators. It is strictly concerned with WETH that has been deposited and is waiting to
     /// be staked.
     uint256 public depositedWethAccountedFor;
+    mapping(address => bool) public consolidationContracts;
 
-    bytes32 public consolidationLastPubKeyHash;
-    address public consolidationSourceStrategy;
-    mapping(address => bool) public consolidationSourceStrategies;
-
-    // For future use
-    uint256[50] private __gap;
+    /// @notice Mapping of the validator hashed keys to balances at last verification.
+    /// Once all the Validators have been consolidated from the old strategy contract this field can be
+    /// deprecated
+    mapping(bytes32 => uint256) public validatorBalances;
```

Comment:
Could this also live in the ConsolidateValidator contract?

Reply:
indeed